src/modules/list.js

```diff
@@ -0,0 +1,144 @@
+export default class Task {
```

Comment:
### It would be better if You can add comments inside your code in your future commits when it goes long and more complex 😉